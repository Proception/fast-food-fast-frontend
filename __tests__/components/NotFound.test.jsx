import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../src/components/NotFound.jsx';

let wrapper;
beforeEach(() => {
  const path = { pathname: 'wrongroute' };
  wrapper = shallow(<NotFound location={path}/>);
});
describe('NotFound Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
