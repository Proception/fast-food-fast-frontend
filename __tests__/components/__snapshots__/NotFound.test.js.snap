// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should match snapshot 1`] = `ShallowWrapper {}`;
